specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -549,18 +541,9 @@
           "description": "The time interval, in minutes, at which the LCM checks a pull service to get updated configurations. This value is ignored if the LCM is not configured in pull mode. The default value is 30."
```

Comment:
(This comment belongs to the line above, I cannot leave comment on the unchanged lines unfortunately)
We need to remove the `default` here because this property is also marked by `readonly`.
This is violating the semantic check.

Reply:
Fixed. Removed the readonly field. This was by mistake.